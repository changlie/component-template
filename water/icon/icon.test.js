// import Vue from 'vue';
import { mount } from 'vue-test-utils';
import Icon from './Icon';

describe('Icon.vue', () => {
  let wrapper = null;

  beforeEach(() => {
    wrapper = mount(Icon);
  });

  it("updates the .textContent to '这是一个图标组件。'", (done) => {
    wrapper.vm.$nextTick(() => {
      try {
        expect(wrapper.text()).toBe('这是一个图标组件。');
        done();
      } catch (err) {
        done.fail(err);
      }
    });
  });
});
