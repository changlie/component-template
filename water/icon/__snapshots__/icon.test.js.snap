// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon.vue 检测快照是否一样。 1`] = `"这是一个图标组件。"`;
